src/runtime-rs/crates/resource/src/network/endpoint/veth_endpoint.rs

```diff
@@ -81,4 +82,7 @@ impl Endpoint for VethEndpoint {
             .context("remove device")?;
         Ok(())
     }
+    async fn save(&self) -> Option<EndpointState> {
+        todo!()
```

Comment:
Again, it would be helpful to put a GH issue URL here so we don't forget and if a user gets a panic, they know where to go :)

Reply:
fixed